totes

```diff
@@ -29,22 +38,34 @@ get_note_to_render() {
       exit 1
     fi
 
+    suppressed_files_registry_spec="$(get_suppressed_files_registry_spec)"
+    if ! [[ -f "$suppressed_files_registry_spec" ]]; then
+      # the suppressed_file_registry must not be empty
+      echo "dummy-text-required-here" > "$suppressed_files_registry_spec"
+    fi
+
     # ref - https://github.com/koalaman/shellcheck/issues/1247
     # shellcheck disable=2012
```

Comment:
this disable and ref no longer seems to apply based on the linter

> after reading [the lint](https://github.com/koalaman/shellcheck/wiki/SC2012) I'm not exactly sure why but  🤷‍♂

Reply:
interesting.  Removed.